mkosi/__init__.py

```diff
@@ -2077,10 +2079,32 @@ def build_uki(
     cmd += ["build", "--linux", kimg]
     mounts += [Mount(kimg, kimg, ro=True)]
 
+    if microcodes:
+        logging.debug("microcode codepath YES")
+        # might need modification
+        initrds = initrds.copy()
```

Comment:
Why the copy here?

Reply:
IIRC the previous version manipulated the input struct (instead of assigning a new list), so that messed with the caller's datastructure. I think this is no longer the case, I'll double check